benefits/eligibility/forms.py

```diff
@@ -6,12 +6,29 @@
 from django import forms
 from django.utils.translation import gettext_lazy as _
 
-from benefits.core import recaptcha, widgets
+from benefits.core import models, recaptcha, widgets
 
 
 logger = logging.getLogger(__name__)
 
 
+class EligibilityVerifierSelectionForm(forms.Form):
+    """Form to capture eligibility verifier selection."""
+
+    action_url = "eligibility:index"
+    method = "POST"
+
+    verifier = forms.ChoiceField(label="", widget=forms.RadioSelect)
+
+    submit_value = _("eligibility.form.continue")
+    submitting_value = _("eligibility.form.continuing")
```

Comment:
See my other comment - there aren't two states for this form. Can just use the same `msgid` here.